### DISCUSSION TITLE
為了自己，要學會拒絕

### DISCUSSION THREAD

**Discussion by Hsucuchia** [1101-GS4719-C/Python-Community-GS4719-C#1283] [General]
拒絕別人真的很困難。我們都希望被喜歡，也不想讓別人失望，所以經常不好意思拒絕。但我們的時間和體力都是有限的，不可能滿足所有人的需求。下次有人找你幫忙時，應該先考慮自己，衡量自己是不是有時間以及心力可以幫助，如果你想要拒絕，可以試試看:
1. 溫和及堅定的語氣
2. 簡單說明自己無法幫忙的原因
3. 提出可能解決的方法
EX: 不好意思，那天我已經有約了，還是你問問看XXX能不能幫忙?
為了自己你要學會拒絕，賈伯斯曾說 : 學會說不，你才能專注在真正重要的事上。
每個人都有自己的事情要完成，相信對方也會理解，所以不要再害怕拒絕別人~

**Comment by Chienyuuu6** [comment]
我超不會拒絕別人，然後又是蠻有責任感的人，常常別人委託我都不好意思拒絕，然後又很多事情要做搞得自己忙到發瘋⋯⋯

不過長大以後開始認知要多愛自己一點，不要把整個人都奉獻給外面的世界，適時評估事情的負荷量是不是會壓迫自己喘不過氣！

懂得拒絕並不是一件壞事，換位思考：別人拒絕我我也不會覺得很糟糕，所以我也可以適時的拒絕不適合自己的邀請！

**Comment by tower20021202** [comment]
我以前也很不會拒絕別人，只要別人需要幫忙，我常常優先幫助他人造成自身的許多事情嚴重delay或是無法完成，但到了現在，在我需要要幫助他人時，我都會先評估好自身的能力及時間是否允許，若不允許便會告訴他原因為何沒班法幫助到他

**Reply by a3240281370** [reply]
你也常常找義安幫忙，還有你打錯字

**Comment by poweichen00** [comment]
我覺得我算是會拒絕別人的，我在答應別人前會先評估會不會影響到我的事情，不會到時候自己的事處理不完，答應別人的事也完成不了，造成兩頭燒。

**Comment by KevinCheng42** [comment]
我覺得學會拒絕他人真的很重要，以前的我不太會拒絕別人，常常累積很多代辦事項而且做不完，現在在回覆他人的邀約前都會先想想自己時間上的安排，適時拒絕，才不會無法好好完成每一件事。

**Comment by Jack-1D** [comment]
很多人都會因為不好意思拒絕而過度擔負責任，這不僅對自己造成壓力，也可能對別人造成不必要的期望和失望。學會拒絕是保護自己和他人的一種方式，而且可以幫助你更好地分配時間和精力，專注於自己真正關心和重要的事情上。除了你提到的溫和且堅定的語氣、簡單解釋自己無法幫忙的原因和提出可能解決的方法，我認為還有其他的技巧可以更好地拒絕別人。例如，可以事先設定自己的優先事項和時間表，以便更清楚地知道何時可以幫助別人，何時需要拒絕。此外，也可以直接說明自己的限制和能力，讓對方更好地了解自己的情況。總之，學會拒絕是一種健康的行為和良好的溝通技巧，可以使我們更好地管理時間和精力，同時還能建立更好的人際關係。

**Comment by harrycoconut** [comment]
我認為我們會怕拒絕別人是因為我們太在乎對方感受，在乎對方感受並沒有不好，但過度在乎導致自己一直無法拒絕別人的話就不好了。說實在一點，其實大家隨時都只注意著自己，你會記得上次別人在你面前跌倒是什麼時候嗎？不不不，你只會記得自己今天早上在跆拳課上滑了一跤，然後快速起身，深怕被別人看到😌

**Comment by llh26** [comment]
我完完全全是一個不會拒絕別人的人，超級在意我的拒絕會對別人造成什麼不便的影響。被朋友開導後，我認為每個人都有自己的事情要做，我不該為了別人的事去犧牲自己處理事情的時間，造成自己的困擾。我覺得答應任何事情前，一定要先衡量自己的能力，不然答應完又沒做好，對別人更不好意思。之後也會參考文章作者提供的方法去試試的！

**Comment by zilinng** [comment]
我以前也超級不會拒絕，甚至因為不會拒絕買了路邊那種騙人超爆貴的愛心筆...但愈長大愈覺得自己這樣真的不行，這真的會給自己的生活帶來太多困擾了，所以學著嘗試回絕自己不願意的事情，有一個小發現，就是簡單明瞭的表達自己不方便其實就能成功拒絕很多事情了，如果太扭捏表達的太婉轉反而別人會容易get不到，最後會以拒絕失敗收場。

**Comment by CCello** [comment]
我不確定我會不會拒絕，但是如果遇到需要拒絕別人的時候我會解釋原因並提供其他幫助的方式，即使你不能幫助對方，也可以提供一些資源，我認為只要你有禮貌、真誠地回應對方，你可以保持你的尊嚴和關係，對方也會理解你的難處。

**Comment by YoYooHuang** [comment]
我覺得人是為了自己而活，沒有必要強迫自己做不喜歡的事情，當有人對你提出不合理或是自己無法接受的要求是，果斷拒絕才是上策。不該是傻傻的聽從別人的意見，這樣只會讓自己更不開心，我覺得拒絕這件事是為了自己，不是為任何人。

**Comment by pigeon9975** [comment]
拒絕別人真的很重要，從以前到現在我都不太好意思拒絕別人，於是身上就堆了好多原本不屬於自己的工作，然後把自己累垮，謝謝你的分享，希望我能改掉這個壞習慣，學會拒絕別人，好好守護自己的肝QQ

**Comment by GuanYann** [comment]
學會拒絕別人是一個必要的技能，因為我們的時間和能量是有限的。拒絕時，應保持禮貌和堅定的態度，提供其他可能的解決方案。這可以幫助我們專注於自己的目標和任務，同時保持彼此之間的關係良好。

**Comment by orianna901001** [comment]
小時候的我是一個很不會拒絕的人，我跟我朋友甚至有幫過倒在麥當勞外面的阿婆買過檸檬紅茶，就因為她想喝(哈)，但我覺得長大之後，還是有慢慢的改變自己，拒絕別人有時候也是一種對自己的美德！

**Comment by hardtagottaname** [comment]
每次拒絕別人時，看到別人失落的表情就會覺得是不是自己錯了，認為不應該讓他人失望、難過，因此陷入無限自責，卻從未想過自己並沒有義務要幫助別人，也不需要因為別人讓自己的情緒低落。我認為，我也要開始學著如何委婉拒絕，遵循自己內心想法，不被他人左右情緒。

**Comment by shuxian12** [comment]
我覺得學會拒絕是一件很重要的事，但有時候對方可能會開始情緒勒索，說我之前也沒有拒絕你幫你幹嘛幹嘛之類的，感覺你非做不可，根本都沒有想到當下的狀況可能不一樣，或是這件事就真的沒辦法做，就會造成爭吵，因此我覺得覺會拒絕勢必很重要，但也希望大家可以學會理解他人的角度，而不是一直主觀的去判斷

**Comment by a3240281370** [comment]
我覺得當我們需要拒絕他人時，重要的是保持尊重和誠實。這些回應傳達了你的無法接受對方的要求，同時保持了禮貌和誠實。

**Comment by Jamesmoneya** [comment]
拒絕別人真的會讓我超級彆扭我常常就會痾痾痾痾痾是可以喇，然後勉強答應他的請求因為根本不會拒絕別人，但是有一些識相的人看到我的身體語言或是語氣就知道我可能沒辦法幫這個忙跑去找其他人，這種人我比較喜歡

**Comment by Alexgns** [comment]
了解自己的價值觀、目標和個人需求，這將有助於確定何時需要說「不」，除此之外練習明確、直接而有禮貌地表達自己的意願，無論是拒絕一項要求、邀請或負擔過多的事情也很重要，很多時候因為自己不好意思，可能為自己帶來不必要的困擾

**Comment by ALAN30359704** [comment]
我認為我是一個不太會拒絕的人，常常有不熟的人突然問我作業或功課，我就會不知道怎麼拒絕的回覆他，尤其是在期中或期末考的時候就覺得非常的困擾因為我也非常的繁忙，而這時大家又是問作業的時刻，所以我認為學會拒絕他人真的是件非常重要的事情！

**Comment by zephyrxin01** [comment]
我覺得拒絕他人是一門藝術，怎樣能讓自己全身而退又不會留下負面的評價很不容易耶。而且當下真的很容易慌張，一旦不小心答應了就麻煩大了。但我也蠻認同前面同學但觀點，也許對方並沒有想那麼多，那我們在拒絕他人但時候也可以試著不要想太多，遵從自己內心想的就好了。

**Comment by youtinglin** [comment]
極度認同真的要學會拒絕，我是一個曾經不捨得拒絕別人的人，總是想我有能力就要盡可能地幫助他人，但是時間久了，漸漸地就變成濫好人或工具人，因此我覺得社會還是很現實的，要思考自己能不能應付，會不會反而造成自己的困擾，如果真的會不舒服就要拒絕，或者是會不會其實被利用了，還是要稍微思考一下利弊，雖然好像有點利己，但至少不要造成最後受傷的是自己。而要如何拒絕?拒絕的當下要清楚表明自己的立場，不能太模稜兩可的答案，立場堅定，如果是朋友的話能理解你的苦衷，但如果是較不熟識的人，也就不用太在意，但前提是說話態度仍是要善意且有禮貌的。

**Comment by alstinlin** [comment]
常常會覺得別人看起來很忙，所以想說順手幫忙一下，結果最後反而讓自己很累，而且幫了一次，之後更不好拒絕第二次，所以我覺得學會拒絕對我來說很重要，也要懂得評估自己是否有能力在不影響自身的情況下，對別人伸出援手

**Comment by 1104030360** [comment]
以前的我也是不太會拒絕別人，但現在的我會考量到如果我沒有能力卻不拒絕可能會帶給別人更大的困擾，所以只要我沒能力時我都會堅定的拒絕，這樣拒絕幫助別人也不會有罪惡感了。

**Comment by xiang305** [comment]
如果一昧地接受別人的請求，他或許不會覺得你人很好，反而會認為你好欺負。我認為適時的拒絕能讓雙方了解彼此的底線，讓日後的相處能夠更加融洽。我這邊也推薦一本我之前讀過的書《你的善良必須有點鋒芒》，裡面大概描述了如何去面對自己真實的想法，走出屬於自己的道路，不再成為任人擺佈的魁儡。最後分享一句這本書裡面的話，「聰明是一種天賦，善良是一種選擇，而後者明顯比前者難」

**Comment by Aaron0810** [comment]
我自己是屬於會拒絕別人的人，但我身邊有個朋友就不是，他完全就是那種會被別人牽著鼻子走的那種，這種人雖然剛開始相處起來很輕鬆，但在一起久了才會發現他的個性，這時候常常會在想究竟他是真的願意還是不好意思拒絕而已，所以每次都還要跟他確定他是不是真的願意，感覺應該要把這篇文分享給他看的。

**Comment by Elaine2899** [comment]
我屬於會拒絕別人的人，但通常只有我真的不想或無法才會拒絕別人。但如果別人情勒我的話我就很難拒絕別人，因此我認為我應該學會杜絕情勒。

**Comment by Evening218** [comment]
我應該算是可以直接拒絕別人的人,但是有時候會為了整體狀況或是避免麻煩選擇接受,我覺得無論拒不拒絕,都應該是經過考慮之後做的決定,而不是一味接受事後再抱怨,可以為了別人犧牲自己一部份的權益,但是也要清楚人與人需要的是互相而不是單方面地索取或給予
